static/locales/ur.yaml

```
@@ -0,0 +1,790 @@
+  'Your Subscription list is currently empty. Start adding subscriptions to see them here.': 'آپ کا
+     رکنیت کی فہرست فی الحال خالی ہے۔ انہیں یہاں دیکھنے کے لیے سبسکرپشنز شامل کرنا شروع کریں۔'
+  'Getting Subscriptions. Please wait.': 'سبسکرپشنز حاصل کرنا۔ برائے مہربانی انتظار کریں.' 
+  Refresh Subscriptions: 'سبسکرپشنز کو ریفریش کریں۔'
+  Load More Videos: 'مزید ویڈیوز لوڈ کریں۔'
+More: 'مزید'
+Channels:
+  Channels: 'چینلز'
+  Title: 'چینل کی فہرست'
+  Search bar placeholder: 'چینلز تلاش کریں۔'
+  Count: '$چینل ملا۔'
+  Empty: 'آپ کے چینل کی فہرست فی الحال خالی ہے۔'
+  Unsubscribe: 'ان سبسکرائب کریں۔'
+  Unsubscribed: '$ کو آپ کی سبسکرپشنز سے ہٹا دیا گیا ہے۔'
+  Unsubscribe Prompt: 'کیا آپ واقعی "$" سے ان سبسکرائب کرنا چاہتے ہیں؟'
+Trending:
+  Trending: ''
+  Default: ''
+  Music: ''
+  Gaming: ''
+  Movies: ''
+  Trending Tabs: ''
+Most Popular: ''
+Playlists: ''
+User Playlists:
+  Your Playlists: ''
+  Playlist Message: 'یہ صفحہ مکمل طور پر کام کرنے والی پلے لسٹس کا عکاس نہیں ہے۔ یہ صرف
+     ان ویڈیوز کی فہرست بناتا ہے جنہیں آپ نے محفوظ کیا ہے یا پسند کیا ہے۔ جب کام ختم ہو جائے تو سب
+     فی الحال یہاں موجود ویڈیوز کو 'پسندیدہ' پلے لسٹ میں منتقل کر دیا جائے گا۔'
+  Your saved videos are empty. Click on the save button on the corner of a video to have it listed here: ''
+  Empty Search Message: 'اس پلے لسٹ میں کوئی ایسی ویڈیوز نہیں ہیں جو آپ کی تلاش سے مماثل ہوں۔'
+  Search bar placeholder: 'پلے لسٹ میں تلاش کریں۔'
+History:
+  # On History Page
+  History: 'تاریخ'
+  Watch History: 'تاریخ دیکھیں'
+  Your history list is currently empty.: 'آپ کی تاریخ کی فہرست فی الحال خالی ہے۔'
+  Empty Search Message: 'آپ کی سرگزشت میں کوئی ایسی ویڈیوز نہیں ہیں جو آپ کی تلاش سے مماثل ہوں۔'
+  Search bar placeholder: "تاریخ میں تلاش کریں۔"
+Settings:
+  # On Settings Page
+  Settings: 
+  The app needs to restart for changes to take effect. Restart and apply change?: 'دی
+     تبدیلیاں اثر انداز ہونے کے لیے ایپ کو دوبارہ شروع کرنے کی ضرورت ہے۔ دوبارہ شروع کریں اور تبدیلی کا اطلاق کریں؟'
+  General Settings: 'ترتیبات'
+    General Settings: 'عام ترتیبات'
+    Check for Updates: 'اپ ڈیٹ کے لیے چیک کریں'
+    Check for Latest Blog Posts: 'تازہ ترین بلاگ پوسٹس چیک کریں۔'
+    Fallback to Non-Preferred Backend on Failure: 'غیر ترجیحی بیک اینڈ پر فال بیک
+       ناکامی پر'
+    Enable Search Suggestions: 'تلاش کی تجاویز کو فعال کریں۔'
+    Default Landing Page: 'ڈیفالٹ لینڈنگ پیج'
+    Locale Preference: ''
+    System Default: ''
+    Preferred API Backend:
+      Preferred API Backend: ''
+      Local API: ''
+      Invidious API: ''
+    Video View Type:
+      Video View Type: ''
+      Grid: ''
+      List: ''
+    Thumbnail Preference:
+      Thumbnail Preference: ''
+      Default: ''
+      Beginning: ''
+      Middle: ''
+      End: ''
+    Current Invidious Instance: ''
+    # $ is replaced with the default Invidious instance
+    The currently set default instance is $: ''
+    No default instance has been set:''
```

a colon and a space(": ") should separate keys from values 
```suggestion
    No default instance has been set: ''
```